hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/FairSchedulerAppsBlock.java

```diff
@@ -128,8 +128,15 @@ protected Boolean hasAccess(RMApp app, HttpServletRequest hsr) {
     }
     return true;
   }
-
-
+  private long appInfoFunc;
+  public String printData(long appInfoFunc) {
+    this.appInfoFunc=appInfoFunc;
```

Comment:
Why are we saving this? We are not using it later. Were you trying to cache here?

Reply:
Was doing this to resolve the error : 'appInfoFunc' hides a field. [HiddenField]